python/pylibcudf/pylibcudf/join.pyx

```diff
@@ -43,14 +47,16 @@ cdef Column _column_from_gather_map(cpp_join.gather_map_type gather_map):
                 device_buffer(),
```

Comment:
I suppose if we knew the size of the buffer needed, you could also pass the `stream.view()` here too?

Reply:
[This column constructor](https://github.com/rapidsai/cudf/blob/05c2bca3f82527e2145a95366523310613c1b876/cpp/include/cudf/column/column.hpp#L86) doesn't take a stream because there is no GPU operation. Is that what you're asking about, or am I misunderstanding? If there's another place we could be passing a stream that I'm missing I'm happy to add that in a follow-up.